api/tests/unit/domain/services/scorecard-service_test.js

```diff
@@ -198,14 +198,14 @@ describe('Unit | Service | ScorecardService', function() {
         const skill = domainBuilder.buildSkill({ id: skillId });
         const targetProfile = domainBuilder.buildTargetProfile({ skills: [skill] });
         campaign = domainBuilder.buildCampaign({ targetProfileId: targetProfile.id, targetProfile });
-        campaignParticipation1 = domainBuilder.buildCampaignParticipation({ campaign, campaignId: campaign.id, isShared: false });
-        campaignParticipation2 = domainBuilder.buildCampaignParticipation({ campaign, campaignId: campaign.id, isShared: false });
-        oldAssessment1 = domainBuilder.buildAssessment({ id: assessmentId1, campaignParticipationId: campaignParticipation1.id });
-        oldAssessment2 = domainBuilder.buildAssessment({ id: assessmentId2, campaignParticipationId: campaignParticipation2.id });
-        oldAssessment1Aborted = domainBuilder.buildAssessment({ ...oldAssessment1, state: Assessment.states.ABORTED });
-        oldAssessment2Aborted = domainBuilder.buildAssessment({ ...oldAssessment2, state: Assessment.states.ABORTED });
-        newAssessment1Saved = domainBuilder.buildAssessment({ id: 67890 });
-        newAssessment2Saved = domainBuilder.buildAssessment({ id: 98760 });
+        campaignParticipation1 = domainBuilder.buildCampaignParticipation({ id: 1, campaign, campaignId: campaign.id, isShared: false });
```

Comment:
@laura-bergoens quel problème ça posait déjà de mettre en dur les ID avec PG ?

Reply:
@lisequesnel Pour le coup ici il n'y a pas de problème, car c'est un test unitaire donc on peut mettre un peu ce qu'on veut. Après, en utilisant le `databaseBuilder` c'est pas non plus que ça pose un problème en soi, mais c'est fortement recommandé de laisser cet outil déterminer l'id de l'enregistrement (car pas de risque de doublon par inadvertance). On peut toujours le mettre soi-même si on est consciencieux.

Reply:
okay, d'ailleurs à ce sujet, j'ai vu qu'il y avait des problèmes dans les seeds. Hors scope de cette PR (of course).